### PR TITLE
Organize blog entries by year

### LINKED ISSUE #140: Setup Blog URL Permalinks
_Originally posted by @chalin in https://github.com/etcd-io/website/pull/112#discussion_r600020880_

There are several ways to set up the blog URL permalinks
- [x] discuss best method for this site
- [x] set `[permalinks]` `blog` variable in `config.toml` based on discussion

### PR DESCRIPTION
**Pull request by chalin**
Closes #140, if you are agreeable to the solution proposed in this PR

Preview: https://deploy-preview-239--etcd.netlify.app/blog/
Redirect test: https://deploy-preview-239--etcd.netlify.app/blog/jepsen-343-results/

/tag docsy
/milestone 21Q2-docsy
/reviewer @nate-double-u

### CODE REVIEW

**Comment by chalin** [comment]
The permalink for a blog entry will be derived from the path to file containing the entry. Hence, for our sole entry, it is `/blog/2020/jepsen-343-results/` for a file found under `/blog/2020/jepsen-343-results.md`. This matches the pattern used by the current site.

To have the last part of the permalink be different from the filename, add a `slug:` entry to the front matter. For example:

```yaml
slug: jepsen-343
```

**Comment by chalin** [comment]
I've added notes to that effect to the front matter of `/blog/_index.md`.

File: config.toml [hunk 1]
```diff
@@ -28,17 +28,6 @@ pygmentsUseClassic = false
...
-# Configure how URLs look like per section.
-[permalinks]
-blog = "/:section/:year/:month/:day/:slug/"
-
-## Configuration for BlackFriday markdown parser: https://github.com/russross/blackfriday
-[blackfriday]
-plainIDAnchors = true
-hrefTargetBlank = true
-angledQuotes = false
-latexDashes = true
```
Comment (chalin) [inline config.toml:40]: We're not using blackfriday.

File: config.toml [hunk 2]
```diff
@@ -28,17 +28,6 @@ pygmentsUseClassic = false
 # See https://help.farbox.com/pygments.html
 pygmentsStyle = "fruity" # "tango"
 
-# Configure how URLs look like per section.
-[permalinks]
-blog = "/:section/:year/:month/:day/:slug/"
```
Comment (chalin) [inline config.toml:33]: We'll go with the default, and so don't need this.

**Review by nate-double-u — APPROVED** [review]

**Pull request merged**